src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1986,6 +1986,8 @@ Total\ items\ found\:=Total items found:
 Selected\ items\:=Selected items:
 Download\ linked\ online\ files=Download linked online files
 Select\ the\ entries\ to\ be\ imported\:=Select the entries to be imported\:
+Entry\ BibTeX\ data\:=Entry BibTeX data\:
```

Comment:
There should not be new translations for this heading. The Entry Editor uses "BibTeX". Please re-use that string. Add the `:` manually in the code (or just style as heading so that no : is necessary).

Reply:
unfortunately there's need for such translation as editor uses:
this.setText(Localization.lang("%0 source", mode.getFormattedName()));
(org.jabref.gui.entryeditor.SourceTab#SourceTab)
But anyways im changing it anyways to just "BibTeX:"

Reply:
Done:
<img width="770" alt="Zrzut ekranu 2024-01-22 o 21 49 23" src="https://github.com/JabRef/jabref/assets/58429636/b3c0163d-f150-47c2-9a69-7e081e0e035e">